catalog-v001.xml

```diff
@@ -1,26 +1,25 @@
 <?xml version="1.0" encoding="UTF-8" standalone="no"?>
 <catalog prefer="public" xmlns="urn:oasis:names:tc:entity:xmlns:xml:catalog">
+    <uri id="Imports Wizard Entry" name="http://emmo.info/emmo/1.0.0-alpha2" uri="emmo.owl"/>
```

Comment:
These changes only makes the catalog file less readable without any real changes. Please exclude the changes to catalog-v001.xml in the PR.